### ISSUE TITLE
Python 3.9 Support

### ISSUE THREAD

**Issue by slerman12** [apple/tensorflow_macos#66]
The latest Python build for M1 Macs and Big Sur is incompatible with this library! Seems odd that this wouldn't be kept aligned with the very Python distribution meant to be used on apple silicon.

**Comment by lokinfey** [comment]
Yes I wait python 3.9.1 build

**Comment by vivek-ram** [comment]
so new python version not support this TensorFlow ??

**Comment by anna-tikhonova** [comment]
> The latest Python build for M1 Macs and Big Sur is incompatible with this library! Seems odd that this wouldn't be kept aligned with the very Python distribution meant to be used on apple silicon.

For this pre-release version, we are using TensorFlow r2.4rc0, which requires Python version 3.6-3.8.

**Comment by MintStudios** [comment]
Well, python 3.8 doesn't have a universal build, only python 3.9+ does.

**Comment by ndom91** [comment]
Yeah so python has officially released a version compatible with M1 Mac OS Big Sur (3.9.1). Any progress on getting this up and running with that? 

I've tried manually messing with the install scripts to change up versions, paths, etc. and wasn't able to get it working.

**Comment by krokhale** [comment]
@ndom91 

I was able to get it working. Switch 3.8 to 3.9 in the install script. Then:

python3 -m venv /Users/krishnarokhale/tensorflow_macos_venv

Once you create the venv. Next run the script again. It should install just fine.

**Comment by iRonJ** [comment]
> @ndom91
> 
> I was able to get it working. Switch 3.8 to 3.9 in the install script. Then:
> 
> python3 -m venv /Users/krishnarokhale/tensorflow_macos_venv
> 
> Once you create the venv. Next run the script again. It should install just fine.

How? I'm getting this error if I try that:

```
ImportError: Python version mismatch: module was compiled for Python 3.8, but the interpreter version is incompatible: 3.9.2 (default, Feb 24 2021, 05:06:40) 
[Clang 12.0.0 (clang-1200.0.32.29)].

```

**Comment by iRonJ** [comment]
I'm having a hard time getting pillow/numpy/matplotlib running natively in python3.8 but these libs runs fine in python3.9, but tensorflow doesn't.
